packages/core/integration-tests/test/macros.js

```diff
@@ -33,19 +34,14 @@ describe('macros', function () {
     });
 
     let res = await overlayFS.readFile(b.getBundles()[0].filePath, 'utf8');
-    assert(res.includes('output=3'));
+    assert(res.includes('output="2a2300bbd7ea6e9a"'));
   });
 
   it('should support renamed imports', async function () {
     await fsFixture(overlayFS, dir)`
       index.js:
-        import { test as foo } from "./macro.js" with { type: "macro" };
-        output = foo(1, 2);
-
-      macro.js:
-        export function test(a, b) {
-          return a + b;
-        }
+        import { hashString as foo } from "@parcel/rust" with { type: "macro" };
```

Comment:
Very nice that you can import any package here, I wouldn't have thought of trying that 👍